manubot/cite/curie/bioregistry.json

```diff
@@ -3275,7 +3353,7 @@
     "name": "Digital Object Identifier",
     "pattern": "^(doi\\:)?\\d{2}\\.\\d{4}.*$",
     "preferred_prefix": "doi",
-    "uri_format": "https://doi.org/$1",
+    "uri_format": "http://dx.doi.org/$1",
```

Comment:
This change seems undesirable.

Reply:
I agree this is undesirable.  But it won't affect any Manubot behavior, right?

Reply:
We could change the tests, but we might have to change them back if bioregistry reverts the format URL. It shouldn't affect Manubot since we use the Manubot DOI handler instead of the CURIE handler.

Reply:
Opened https://github.com/biopragmatics/bioregistry/pull/316 so we can wait on that